sdk/src/main/java/com/bugsnag/android/Stacktrace.java

```diff
@@ -28,7 +28,11 @@ public void toStream(@NonNull JsonStream writer) throws IOException {
             StackTraceElement el = stacktrace[k];
             try {
                 writer.beginObject();
-                writer.name("method").value(el.getClassName() + "." + el.getMethodName());
+                if (el.getClassName().length() > 0) {
```

Comment:
It'd be good to add a unit test for this behaviour, as presumably we've encountered this scenario in the wild?

Reply:
Everyone who tried `bugsnag_notify` saw it; I was surprised no one asked why the method names had a leading `.` or why they didn't demangle. 😅